Android 8.0 crash

Caused by: java.lang.RuntimeException: Cannot get job id from manifest.  Make sure that the immediateScanJobId is configured in the manifest for the ScanJob.
at org.altbeacon.beacon.service.ScanJob.getJobIdFromManifest(ScanJob.java:312)
at org.altbeacon.beacon.service.ScanJob.getImmediateScanJobId(ScanJob.java:282)

version:51c3fff
meta-data android:name="immmediateScanJobId" android:value="208352939"

You add this code, but the word is used wrong.
immmediateScanJobId => immediateScanJobId

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,11 @@
+### 2.13.1 / 2018-03-05
+
+[Full Changelog](https://github.com/AltBeacon/android-beacon-library/compare/2.13.1...2.13)
+
+ Bug Fixes:
+ - Fix incorrect immediateScanJobId meta-data name in manifest causing crash  (#653, David G. Young)
+
+
 ### 2.13 / 2018-03-05
 
 [Full Changelog](https://github.com/AltBeacon/android-beacon-library/compare/2.12.4...2.13)

--- a/src/main/AndroidManifest.xml
+++ b/src/main/AndroidManifest.xml
@@ -30,7 +30,7 @@
 
         <service android:name=".service.ScanJob"
                 android:permission="android.permission.BIND_JOB_SERVICE">
-            <meta-data android:name="immmediateScanJobId" android:value="208352939" />
+            <meta-data android:name="immediateScanJobId" android:value="208352939" />
             <meta-data android:name="periodicScanJobId" android:value="208352940" />
         </service>
   

